src/cdfDatasource.ts

```diff
@@ -52,7 +51,7 @@ export function formQueryForItems(
       granularity: granularity || ms2String(options.intervalMs),
     };
   }
-  const limit = calculateDPLimitPerQuery(isAggregated, items.length);
+  const limit = calculateDPLimitPerQuery(items.length);
```

Comment:
for non-aggregated values in normal timeseries limit is still 100k
so no need to change here